warehouse/ingest-core/src/main/java/datawave/ingest/table/config/AbstractTableConfigHelper.java

```diff
@@ -97,33 +97,18 @@ protected void setAggregatorConfigurationIfNecessary(String tableName, List<Comb
     }
     
     /**
-     * This calls the generateInitialTableProperties on the IteratorUtil class using reflection because they changed the signature between 1.4.1 and 1.4.2 of
-     * ACCUMULO and did not supply any backward compatibility (argh).
-     * 
-     * @return
+     * Copied from Accumulo 1.9 IteratorUtil
      */
     public static Map<String,String> generateInitialTableProperties() {
-        Map<String,String> props = null;
-        try {
-            Method method = IteratorUtil.class.getMethod("generateInitialTableProperties");
-            props = (Map<String,String>) (method.invoke(null));
-        } catch (SecurityException e) {
-            try {
-                Method method = IteratorUtil.class.getMethod("generateInitialTableProperties", boolean.class);
-                props = (Map<String,String>) (method.invoke(null, true));
-            } catch (Exception ex) {
-                throw new RuntimeException("Unable to call generateInitialTableProperties", ex);
-            }
-        } catch (NoSuchMethodException e) {
-            try {
-                Method method = IteratorUtil.class.getMethod("generateInitialTableProperties", boolean.class);
-                props = (Map<String,String>) (method.invoke(null, true));
-            } catch (Exception ex) {
-                throw new RuntimeException("Unable to call generateInitialTableProperties", ex);
-            }
-        } catch (Exception e) {
-            throw new RuntimeException("Unable to call IteratorUtil.generateInitialTableProperties()", e);
+        TreeMap<String,String> props = new TreeMap<>();
+        
+        for (IteratorScope iterScope : IteratorScope.values()) {
+            props.put(Property.TABLE_ITERATOR_PREFIX + iterScope.name() + ".vers", "20,VersioningIterator");
```

Comment:
`VersioningIterator` doesn't have a package name. I'm guessing maybe that's leading to the test errors.